### PR TITLE
Use common transformer

### PR DESCRIPTION
**Pull request by dcdarrell9**
## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by dcdarrell9** [comment]
[(5) Update transformers to use common formatters](https://trello.com/c/bsHlXw2C/288-5-update-transformers-to-use-common-formatters)

File: tests/test_views.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
-from transform.transformers import PDFTransformer, ImageTransformer, CSTransformer
 from transform import app
 from jinja2 import Environment, PackageLoader
 
-from flask import make_response, send_file
+from flask import send_file
```
Comment (JamesGardiner) [inline tests/test_views.py]: Seeing as you are changing the import section, can you make it PEP8 compliant?

**Comment by insacuri** [comment]
The thing I've noticed is that a lot of tests are being deleted and it doesn't look like they're being replaced by anything.  Is that because we're using the common transformer and that's already got unit tests?

**Comment by dcdarrell9** [comment]
@insacuri Yes and no, there are unit tests in common that are being used instead, but I think we could do with more tests, but we do have card for that.

**Pull request merged**